pages/_app.js

```diff
@@ -13,9 +14,12 @@ function MyApp({ Component, pageProps, locale, messages }) {
     cache
   )
   return (
-    <RawIntlProvider value={intl}>
-      <Component {...pageProps} />
-    </RawIntlProvider>
+		<>
+			<GlobalStyle />
+    	<RawIntlProvider value={intl}>
+      	<Component {...pageProps} />
+    	</RawIntlProvider>
+		</>
```

Comment:
In code editor, the formatting looks all good. Weird that it's looking like this here.